### ISSUE TITLE
When scheduling a single execution job, fails to execute

### ISSUE THREAD

**Issue by namack75** [jlinn/quartz-redis-jobstore#21]
Came across the issue when upgrading from 1.1.5 to 1.1.7.

I scheduled single execution jobs, and had no issues with execution with 1.1.5.
After the update, I am now getting exceptions:

```
org.quartz.JobPersistenceException: Could not set triggers as fired.
        at net.joelinn.quartz.jobstore.RedisJobStore.doWithLock(RedisJobStore.java:1128) ~[quartz-redis-jobstore-1.1.7.jar:?]
	at net.joelinn.quartz.jobstore.RedisJobStore.triggersFired(RedisJobStore.java:1051) ~[quartz-redis-jobstore-1.1.7.jar:?]
	at org.quartz.core.QuartzSchedulerThread.run(QuartzSchedulerThread.java:336) [quartz-2.2.3.jar:?]
Caused by: java.lang.NullPointerException
	at net.joelinn.quartz.jobstore.RedisStorage.triggersFired(RedisStorage.java:694) ~[quartz-redis-jobstore-1.1.7.jar:?]
	at net.joelinn.quartz.jobstore.RedisStorage.triggersFired(RedisStorage.java:26) ~[quartz-redis-jobstore-1.1.7.jar:?]
	at net.joelinn.quartz.jobstore.RedisJobStore$41.doWithLock(RedisJobStore.java:1055) ~[quartz-redis-jobstore-1.1.7.jar:?]
	at net.joelinn.quartz.jobstore.RedisJobStore$41.doWithLock(RedisJobStore.java:1051) ~[quartz-redis-jobstore-1.1.7.jar:?]
	at net.joelinn.quartz.jobstore.RedisJobStore.doWithLock(RedisJobStore.java:1121) ~[quartz-redis-jobstore-1.1.7.jar:?]
	... 2 more
```

Looks to be related to this change (RedisStorage:694):
```
final long nextFireTime = trigger.getNextFireTime().getTime();
jedis.hset(triggerHashKey, TRIGGER_NEXT_FIRE_TIME, Long.toString(nextFireTime));
setTriggerState(RedisTriggerState.WAITING, (double) nextFireTime, triggerHashKey, jedis);
```

Would it be the case that since its a single execution, that its failing to find a next execute?

**Comment by jlinn** [comment]
Thanks for reporting this.  That was definitely the cause.  The bug should be fixed in [v1.1.8](https://github.com/jlinn/quartz-redis-jobstore/releases/tag/quartz-redis-jobstore-1.1.8).

**Comment by namack75** [comment]
Yup. Looks like its working. THanks.
